packages/yoroi-extension/chrome/extension/background.js

```diff
@@ -1433,6 +1434,49 @@ function handleInjectorConnect(port) {
               handleError(e);
             }
           break;
+          case 'list_nfts/cardano':
+            try {
+              await withDb(async (db, localStorageApi) => {
+                await withSelectedWallet(
+                  tabId,
+                  async (wallet) => {
+                    const assets = await connectorGetAssets(wallet);
+                    const potentialNFTAssets = assets.filter(asset => asset.amount === '1');
+                    const fetcher = await getCardanoStateFetcher(localStorageApi);
+                    const metadatasList = await fetcher.getMultiAssetMintMetadata({
```

Comment:
just being nitpicking: "data" is the plural of "datum" so no need to add the "-s" suffix